GitExtUtils/GitUI/UIExtensions.cs

```diff
@@ -43,5 +45,27 @@ public static bool IsFixedWidth(this Font ft, Graphics g)
 
             return fixedWidth;
         }
+
+        public static string FormatBodyAndNotes(string bodyOrSubject, string? notes)
```

Comment:
❔
```suggestion
        internal static string FormatBodyAndNotes(string bodyOrSubject, string? notes)
```

Reply:
Looks pretty "public" to me, it is used by other assemblies not mentioned by "InteralsVisibleToAttribute". Needed to put this code "somewhere", @RussKie left such location choice with no further comments.